core/src/main/java/com/linecorp/armeria/common/ContentDisposition.java

```diff
@@ -64,10 +63,14 @@
  */
 public final class ContentDisposition {
 
-    // Forked from https://github.com/spring-projects/spring-framework/blob/d9ccd618ea9cbf339eb5639d24d5a5fabe8157b5/spring-web/src/main/java/org/springframework/http/ContentDisposition.java
+    // Forked from https://github
+    // .com/spring-projects/spring-framework/blob/d9ccd618ea9cbf339eb5639d24d5a5fabe8157b5/spring-web/src
+    // /main/java/org/springframework/http/ContentDisposition.java
 
-    private static final ContentDisposition EMPTY =
-            new ContentDisposition("", null, null, null, null, null, null, null);
+    private static final ContentDisposition EMPTY = new ContentDisposition("", null, null, null);
+
+    private static final Map<String, Charset> supportedCharsets =
+            ImmutableMap.of("utf-8", UTF_8, "iso-8859-1", ISO_8859_1);
```

Comment:
Shouldn't we support other versions, like all upper- and lower-cased names from `Charset.aliases()`?

Reply:
As per RFC5987, "UTF-8" the "ISO-8859-1" should be used. https://github.com/line/armeria/pull/3253#discussion_r553738233